feature-flags/tests/common.rs

```diff
@@ -0,0 +1,66 @@
+use std::net::SocketAddr;
+use std::str::FromStr;
+use std::string::ToString;
+use std::sync::Arc;
+
+use once_cell::sync::Lazy;
+use rand::distributions::Alphanumeric;
+use rand::Rng;
+use tokio::net::TcpListener;
+use tokio::sync::Notify;
+
+use feature_flags::config::Config;
+use feature_flags::server::serve;
+
+pub static DEFAULT_CONFIG: Lazy<Config> = Lazy::new(|| Config {
+    address: SocketAddr::from_str("127.0.0.1:0").unwrap(),
+    redis_url: "redis://localhost:6379/".to_string(),
+    write_database_url: "postgres://posthog:posthog@localhost:15432/test_database".to_string(),
+    read_database_url: "postgres://posthog:posthog@localhost:15432/test_database".to_string(),
+    max_concurrent_jobs: 1024,
+    max_pg_connections: 100,
+});
+
+pub struct ServerHandle {
+    pub addr: SocketAddr,
+    shutdown: Arc<Notify>,
+}
+
+impl ServerHandle {
```

Comment:
I tried not re-building this and just using axum-test, but ran into annoyances there around the IP extraction not working well, what capture does sans axum-test-helper indeed seems the most flexible + useful here 👍

Reply:
I meant this one: https://docs.rs/axum-test/latest/axum_test/ but indeed control over all the boilerplate here, given its not a lot is 👍